mozilla/fx-private-relay
# Add Phone Masking What's New entry

# New feature description

This adds the new What's New Entry for Phone Masking. 

# Screenshot (if applicable)

https://user-images.githubusercontent.com/13066134/186756783-a69d748c-5a58-424f-a91d-364bdf068e82.mov

<img width="495" alt="image" src="https://user-images.githubusercontent.com/13066134/186512025-3d83c339-540a-4b50-b3bf-22c260f322fd.png">


Non-premium (w/o upgrade CTA):
<img width="501" alt="image" src="https://user-images.githubusercontent.com/13066134/186511850-9682e1d0-a33b-4a40-be31-6cdcf54ecfd2.png">


Premium (w/ upgrade CTA)
<img width="495" alt="image" src="https://user-images.githubusercontent.com/13066134/186511973-2995eba9-bc30-4362-ae31-1a2ed16478e1.png">



# How to test

1. With a non-premium account, check that a new news item appears in the What's New module. The Phone Masking news item should show with an upgrade button.
2. With a premium account, run through the same step as 1. and check that the upgrade button does not show.

# Checklist
